test/versioned/cjs-in-esm/package.json

```diff
@@ -10,12 +9,15 @@
           "node": ">=16.12.0 <20"
```

Comment:
this isn't running on node 20

Reply:
bad copy/paste, thank you for the catch!